packages/sveltekit/src/server/handlers/auth.ts

```diff
@@ -0,0 +1,8 @@
+import type { Handle } from '@sveltejs/kit';
+import { sequence } from '@sveltejs/kit/hooks';
+import { callback } from './callback';
+import { session } from './session';
+
+export function auth(): Handle {
+  return sequence(callback(), session());
```

Comment:
I'm not sure if you can wrap a sequence in a sequence with SvelteKit. The reason I'm saying this is that if someone was to use this function, they wouldn't be able to add any additional function to the sequence if they can't wrap a sequence in a sequence.

Reply:
That shouldn´t be a problem as sequence is basically just a handle function. But i was thinking about removing `auth` anyway since it´s simple to just put `callback` and `session` in there. And some people might prefer not using the callback handle but rather an endpoint.

Reply:
I can see that both `callback()` and `session()` are exported too, so we could keep this nice all in one handler along with the individual ones. Documenting them is what will be key to how developers will use them.